python/test_python.py

```diff
@@ -82,33 +82,40 @@ def test_read_file(self, docker_container):
 
 
 class TestArgs:
+    """Test that args can be passed to script"""
     @pytest.mark.parametrize(
         'docker_container',
         ['python arguments.py "Argument Number 1"'],
         indirect=True,
     )
     def test_args(self, docker_container):
         docker_container.wait()
-        assert docker_container.logs() == b'Argument Number 1\n'
+        assert docker_container.logs() == b'argument number 1\n'
 
 
 class TestReadJsonFile:
+    """Test that a JSON file is read correctly"""
     @pytest.mark.parametrize(
         'docker_container',
         ['python read_json.py < person-records.json'],
         indirect=True,
     )
     def test_read_json_file(self, docker_container):
-
         file_path = "./python/person-records.json"
-
         with open(file_path, "r") as file:
             data = json.load(file)
-
         expected = "".join(f"Hello, {person['age']} year old {person['first_name']}\n" for person in data)
 
         docker_container.wait()
         assert str(docker_container.logs(), 'UTF-8') == expected
 
-# to run in command line:
-# docker run -it --rm rosetta-python
+
+class TestWriteFile:
+    """Test that a script, given a path to a file, can write to that file"""
+    @pytest.mark.parametrize(
+        'docker_container',
+        ['python write_file.py output.txt "Bob Barker"'],
```

Comment:
Let's not leave it up to the script (i.e. `write_file.py`) to report the file's contents because it doesn't prove that the script actually wrote to the file, only that the script upper-cased and printed out the input text.

Instead, we could do something like this:
`['python write_file.py output.txt "Bob Barker"; cat output.txt'],`

For clarity, it might even be worth formatting it like this:
```python
["""
  python write_file.py output.txt "Bob Barker"
  cat output.txt
"""],
```

Reply:
Good point. Makes sense to have a super simple script